monetio/models/hysplit.py

```diff
@@ -339,12 +347,16 @@ def parse_header(self, hdata1):
            number of starting locations in file.
         """
         if len(hdata1["start_loc"]) != 1:
-            print("WARNING in ModelBin _readfile - number of starting locations " "incorrect")
-            print(hdata1["start_loc"])
+            logger.warning(
+                f"WARNING in ModelBin {self.filename} _readfile - number of starting locations incorrect"
```

Comment:
If this is a message intended for the user, suggest using `warnings.warn()` instead of the logger. In either case, the word "warning" doesn't need to be included in the message since that should already be clear.